app/modules/clustersync/TagSyncUpdateProcessor.scala

```diff
@@ -2,30 +2,53 @@ package modules.clustersync
 
 import com.amazonaws.services.kinesis.model.Record
 import com.gu.tagmanagement.{EventType, TagEvent}
-import model.Tag
 import play.api.Logger
-import repositories.TagLookupCache
+import repositories.{TagLookupCache, TagRepository}
 import services.KinesisStreamRecordProcessor
 
-object TagSyncUpdateProcessor extends KinesisStreamRecordProcessor {
+import scala.util.{Failure, Success, Try}
+
+object TagEventDeserialiser {
 
   import services.KinesisRecordPayloadConversions._
 
+  def deserialise(record: Record): Try[TagEvent] = {
+    val tProto = kinesisRecordAsThriftCompactProtocol(record, stripCompressionByte = true)
```

Comment:
Can we add a comment around why we're setting `stripCompressionByte` to true here?

Reply:
sure

Reply:
added